mobile/src/main/java/org/openhab/habdroid/ui/WidgetAdapter.kt

```diff
@@ -1357,15 +1357,20 @@ class WidgetAdapter(
                 return false
             }
 
-            // hide dividers before and after frame widgets
+            // Hide dividers before and after frame widgets
             val adapter = parent.adapter
-            if (adapter != null) {
+            val viewTypeCurrentPos = adapter?.getItemViewType(position)
+            if (viewTypeCurrentPos != null) {
```

Comment:
I think I preferred the previous version, since that new variant is just an obscured null check of `adapter` as well.

Reply:
It saves one `adapter?.getItemViewType(postion)` call. Not sure how expensive this call is.

Reply:
How? AFAICT the variable is used exactly once below.

Reply:
I forgot to use it in https://github.com/openhab/openhab-android/pull/2381/files#diff-b5b8a2ffbcde1bf9caec46b35e310bb0cb1f47171a75e0482527c06680d330f3R1364